lib/features.ts

```diff
@@ -108,6 +108,18 @@ export const Features = {
       "The library has built-in functionality which lets you download a PDF.",
     important: false,
   },
+  imgExport: {
+    title: "image Export",
+    description:
+      "The library has built-in functionality which lets you export image.",
+    important: false,
+  },
+  treeHierarchy: {
+    title: "tree hierarchy",
+    description:
+      "The library has built-in functionality which can show struct data with tree hierarchy.",
+    important: false,
+  },
```

Comment:
I don't think we need these features. PDF and Word export ability seems more important than images. Also, I think the `treeHierarchy` feature you added is a duplicate of `trees` earlier in the file.

Reply:
I have delete this features  @statico